### DISCUSSION TITLE
Fakes players

### DISCUSSION THREAD

**Discussion by admkiller** [azerothcore/azerothcore-wotlk#3966] [General]
I wonder if there is, or does anyone have Fake Players Script or module that appears in WHO.

**Comment by Bogir** [comment]
Place bounty on it 
https://discordapp.com/channels/217589275766685707/546431432424292362

**Comment by Viste** [comment]
https://github.com/Viste/OutlandCore/tree/fake

**Comment by Darkensand** [comment]
> https://github.com/Viste/OutlandCore/tree/fake

There is a kernel compilation error

![Безымянный](https://user-images.githubusercontent.com/569097/71698831-a8155900-2dc5-11ea-8dd1-96c17edd8763.png)


[ 79%] Building CXX object src/server/game/CMakeFiles/game.dir/Instances/InstanceScript.cpp.o
/opt/server/wow/OutlandCore-fake/src/server/game/Handlers/MiscHandler.cpp: In member function ‘void WorldSession::HandleWhoOpcode(WorldPacket&)’:
/opt/server/wow/OutlandCore-fake/src/server/game/Handlers/MiscHandler.cpp:376:92: error: ‘GetAreaEntryByAreaID’ was not declared in this scope
         if (AreaTableEntry const* areaEntry = GetAreaEntryByAreaID(itr->second->GetZoneId()))
                                                                                            ^
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Loot/LootItemStorage.cpp.o
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Loot/LootMgr.cpp.o
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Mails/Mail.cpp.o
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Maps/Map.cpp.o
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Maps/MapInstanced.cpp.o
[ 80%] Building CXX object src/server/game/CMakeFiles/game.dir/Maps/MapManager.cpp.o
src/server/game/CMakeFiles/game.dir/build.make:2630: recipe for target 'src/server/game/CMakeFiles/game.dir/Handlers/MiscHandler.cpp.o' failed
make[2]: *** [src/server/game/CMakeFiles/game.dir/Handlers/MiscHandler.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:862: recipe for target 'src/server/game/CMakeFiles/game.dir/all' failed
make[1]: *** [src/server/game/CMakeFiles/game.dir/all] Error 2
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2

**Comment by Darkensand** [comment]
> https://github.com/Viste/OutlandCore/tree/fake

Можешь сделать под новую версию?*

**Comment by Bogir** [comment]
> Можешь сделать под новую версию?*

Чего сделать? Выдёргивай себе скрипты отсюда

**Comment by Darkensand** [comment]
> > Можешь сделать под новую версию?*
> 
> Чего сделать? Выдёргивай себе скрипты отсюда

дак там ошибка
https://user-images.githubusercontent.com/569097/71698831-a8155900-2dc5-11ea-8dd1-96c17edd8763.png

**Comment by Bogir** [comment]
Дак тебе готовый, на блюдечке вынь да полож?
![изображение](https://user-images.githubusercontent.com/25766571/71730808-3801e400-2e54-11ea-851a-6cd8b1127685.png)
Открой эти исходники и выдерни из них себе скрипты, добейся, чтоб работали и наслаждайся результатом
